pyproject.toml

```diff
@@ -194,3 +194,13 @@ source = [
 # NBQA CONFIGURATION                                                          #
 [tool.nbqa.addopts]
 ruff = ["--ignore=E402"]
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# LINK CHECKER CONFIGURATION                                                  #
+[tool.md_dead_link_check]
+exclude_links = [
+    "https://github.com/openvinotoolkit/anomalib/settings/actions/runners/new",
+    "https://openvinotoolkit.github.io/anomalib/*", # Remove it after publish documentation
+]
+exclude_files = ["docs/*"]
+force_get_requests_for_links = ["https://secure.ethicspoint.com/*"]
```

Comment:
What's this part?

Reply:
The link exists in code of conduct. https://github.com/openvinotoolkit/anomalib/blob/5babb3b39290c66cf5d8210f8cae2ab1f8825ea2/CODE_OF_CONDUCT.md?plain=1#L63
This site is not supported `head` request, always return 404, need to use get request to check link.